src/extension/commands_helpers.c

```diff
@@ -455,3 +500,31 @@ static void _dump_out_msg(dd_log_level_t lvl, zend_llist *iovecs)
         zend_string_release(zstr);
     }
 }
+
+dd_result dd_command_process_config_features(
+    mpack_node_t root, ATTR_UNUSED void *nullable ctx)
+{
+    UNUSED(ctx);
+    mpack_node_t first_element = mpack_node_array_at(root, 0);
+    bool new_status = mpack_node_bool(first_element);
+
+    if (DDAPPSEC_G(enabled_by_configuration) == ENABLED && !new_status) {
+        DDAPPSEC_G(enabled) = ENABLED; // Configuration dictates
+        mlog(dd_log_debug, "Remote config is trying to disable extension but "
+                           "it is enabled by config");
+        return dd_error;
+    }
+    DDAPPSEC_G(enabled) = new_status ? ENABLED : DISABLED;
+    return dd_success;
+}
+
+dd_result dd_command_process_config_features_unexpected(
+    mpack_node_t root, ATTR_UNUSED void *nullable ctx)
+{
+    UNUSED(root);
+    UNUSED(ctx);
+    mlog(dd_log_debug,
+        "Config_features response was given to an unexpected request");
```

Comment:
```suggestion
        "Unexpected config_features response to request");
```

The request isn't unexpected, it's the config_features.

Reply:
Done